repos/sourcelink.proj

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PreferredHash>51310e65e75010467993f793e1739d1a1dad50c5</PreferredHash>
+    <SourceDirectory>sourcelink</SourceDirectory>
+  </PropertyGroup>
+
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
+
+  <PropertyGroup>
+    <OfficialBuildId>20190623.1</OfficialBuildId>
+
+    <BuildCommandArgs/>
+    <BuildCommandArgs>$(BuildCommandArgs) --pack</BuildCommandArgs>
+    <BuildCommandArgs>$(BuildCommandArgs) --configuration $(Configuration)</BuildCommandArgs>
+    <BuildCommandArgs>$(BuildCommandArgs) --binaryLog</BuildCommandArgs>
+    <BuildCommand>$(ProjectDirectory)build$(ShellExtension) $(BuildCommandArgs)</BuildCommand>
+
+    <GlobalJsonFile>$(ProjectDirectory)global.json</GlobalJsonFile>
+    <NuGetConfigFile>$(ProjectDirectory)/NuGet.config</NuGetConfigFile>
+    <LogVerbosityOptOut>true</LogVerbosityOptOut>
+    <OutputPlacementRepoApiImplemented>false</OutputPlacementRepoApiImplemented>
+    <PackagesOutput>$(ProjectDirectory)artifacts/packages/$(Configuration)/Shipping/</PackagesOutput>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <RepositoryReference Include="arcade" />
+    <RepositoryReference Include="xliff-tasks" />
+    <RepositoryReference Include="newtonsoft-json" />
+    <RepositoryReference Include="newtonsoft-json901" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <!-- reset SDK location variables to use 3.0 SDK override -->
+    <EnvironmentVariables Remove="DOTNET_TOOL_DIR=$(DotNetCliToolDir)" />
+    <EnvironmentVariables Remove="DotNetCoreSdkDir=$(DotNetCliToolDir)" />
+    <EnvironmentVariables Remove="DotNetRoot=$(DotNetCliToolDir)" />
+    <EnvironmentVariables Remove="DotNetToolPath=$(DotNetCliToolDir)" />
+    <EnvironmentVariables Remove="DotNetTool=$(DotNetCliToolDir)dotnet" />
+    <EnvironmentVariables Remove="DOTNET_HOST_PATH=$(DotNetCliToolDir)dotnet" />
+    <EnvironmentVariables Remove="_InitializeDotNetCli=$(DotNetCliToolDir)" />
+
+    <EnvironmentVariables Include="DOTNET_TOOL_DIR=$(DotNet3CliToolDir)" />
```

Comment:
The removes then includes look very brittle to me, I'd expect switching between CLIs to be implemented in `dir.props`/`dir.targets`. If this is copy-paste from an existing place it's implemented this way, though, maybe fixing this shouldn't block the PR.

Reply:
I utilized the source for sourcelink.proj from xdt.proj, if this seems redundant I can go ahead and remove this.

Reply:
Ok, it doesn't appear redundant, just a fragile implementation. It would take some extra work to rework both places so I don't think it needs to be improved in this PR.

Reply:
Yeah, at this point we should be defaulting to the 3.0 SDK and making the exception the 2.x one if it's still needed.  This should happen in a different PR, I logged https://github.com/dotnet/source-build/issues/1129 for this.